GetComponents

```diff
@@ -156,6 +156,10 @@ GetOptions(
 pod2usage(1) if $HELP;
 pod2usage( -verbose => 2 ) if $MAN;
 
+if ( $PARALLEL && !defined($QUEUE) ) {
+    &DIE("--parallel is not supported for this Perl installation.");
```

Comment:
Please tell them what they need to install in order to make it work.

Reply:
I was thinking about that. But: on a cluster? I do not really want to encourage email of the form "Dear Cluster admin, you must install for me 'apt-get install perl' so that I can compile the Einstein Toolkit!".  The only other place this ever happened and we know of was on a Docker image where only (the equivalent of) \[perl-base\](https://packages.debian.org/bookworm/perl-base) was installed.

The problem will (maybe, old documentation apparently being used long after it gets marked with red blinking headers of "donotuuse") go away when we remove the `--parallel` and `--shallow` explicit options from all download instructions (since those options are defaults now anyway and the automatic parallel has been checking for Thread::Queue for a while.).  
  
We really don't need parallel checkouts that much anymore (since there are many fewer repos to check out with git than with svn).   
  
I can add "On your own system you can avoid this by installing a full 'perl' package using your package manager.".

Reply:
How about this?

"Error: Parallel checkout is not possible unless the Perl packages threads, threads::shared, Thread::Queue;, and Scalar::Util are installed.  Please either (a) install these packages, or (b) do not use the normal serial checkout."

Reply:
updated to read:  
 

```perl
if ( $PARALLEL && !defined($QUEUE) ) {
    &DIE("Parallel checkout is not possible unless the Perl packages threads, threads::shared, Thread::Queue, and Scalar::Util are installed. Please either (a) install these packages, or (b) leave out the --parallel option.");
}
```